content/millennium/r4/financial/general/insurance-plan.md

```diff
@@ -0,0 +1,67 @@
+---
+title: InsurancePlan | R4 API
+---
+
+# InsurancePlan
+
+* TOC
+  {:toc}
```

Comment:
Is there extra spaces before toc that is causing it not to populate in the screenshot?